_summaries/kingma2014adam_1.md

```diff
@@ -3,17 +3,39 @@ layout: summary
 title: Summary
 paper: {{paper_tag}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: jordi1215
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
```

Comment:
It might not be very important, but I think it is good to list some similar methods e.g., Nadam and AMSGrad. Even by now, there are several papers that try to propose a better adam-like function, thus I personally find it interesting to compare the differences between each of them.